README.md

```diff
@@ -8,33 +8,65 @@
 A plugin bringing remote execution to the Foreman, completing the config 
 management functionality with remote management functionality.
 
+* Website: [theforeman.org](http://theforeman.org)
+* ServerFault tag: [Foreman](http://serverfault.com/questions/tagged/foreman)
+* Issues: [foreman_remote_execution Redmine](http://projects.theforeman.org/projects/foreman_remote_execution/issues)
+* Wiki: [Foreman wiki](http://projects.theforeman.org/projects/foreman/wiki/About)
+* Community and support: [#theforeman](https://kiwiirc.com/client/irc.freenode.net/?#theforeman) for general support, [#theforeman-dev](https://kiwiirc.com/client/irc.freenode.net/?#theforeman-dev) for development chat in [Freenode](irc.freenode.net)
+* Mailing lists:
+    * [foreman-users](https://groups.google.com/forum/?fromgroups#!forum/foreman-users)
+    * [foreman-dev](https://groups.google.com/forum/?fromgroups#!forum/foreman-dev)
+  
+## Features
+
+* Visualize remote execution job process live 
+![job detail](http://theforeman.org/plugins/foreman_remote_execution/0.0/job_detail_1.png)
+* Schedule or run jobs on hosts 
+![invocation form](http://theforeman.org/plugins/foreman_remote_execution/0.0/invocation_form.png)
+* Create templates to customize your jobs
+![job templates](http://theforeman.org/plugins/foreman_remote_execution/0.0/job_template_form.png)
+
 ## Installation
 
-See [How_to_Install_a_Plugin](http://projects.theforeman.org/projects/foreman/wiki/How_to_Install_a_Plugin)
-for how to install Foreman plugins
+Please see the Foreman manual for appropriate instructions:
 
-## Usage
+* [Foreman: How to Install a Plugin](http://theforeman.org/plugins/)
```

Comment:
Right, I meant the install instructions don't belong here at all. As given, they're not really even enough to use the plugin, the manual has the full instructions and because things are moving quickly I think we should just have them in one place.